src/pages/kb/user-guide/users/authentication-options.md

```diff
@@ -10,61 +10,96 @@ Authentication settings are available in the `Settings`>`Settings` tab.
-2. Google SSO: add Google Apps based domains to allow login from. Any user with a Google account from these domain(s) will be able to login. If they don't have an account yet, an account will be automatically created.
+2. Google SSO: add Google Apps based domains to allow login from. Any user with
+   a Google account from these domain(s) will be able to login. If they don't
+   have an account yet, an account will be automatically created.
 3. Enable/disable SAML support.
 
-{% callout warning %}
-Please note that SAML authentication is no longer available for Hosted Redash customers. Existing configurations are not affected.
+{% callout warning %} Please note that SAML authentication is no longer
+available for Hosted Redash customers. Existing configurations are not affected.
 {% endcallout %}
 
 Once you enable SAML, additional settings will appear:
 
 ![](/assets/images/docs/gitbook/saml-details.png)
 
-1. SAML Metadata URL is required to make SAML authentication work. You can get this URL from your SAML provider like [OneLogin](https://www.onelogin.com/connector/redash).
-2. Set up Redash callback URL in your SAML provider: `https://app.redash.io/saml/callback?org_slug=<organization name>`. If you have single organization name, set `default` as value for org_slug.
-3. Within SAML provider, map user fields FirstName and LastName to matching values as those two fields are mandatory for user creation.
-
-By default any user created with SAML/SSO will join the default group. It's possible to configure the SAML provider to pass what groups the user should join by setting the `RedashGroups` parameter. If you use OneLogin's predefined Redash application, it will always pass this parameter, meaning that even for existing users, it will override their current groups memberships. Hence you need to make sure it's up to date.
+1. SAML Metadata URL is required to make SAML authentication work. You can get
+   this URL from your SAML provider like
+   [OneLogin](https://www.onelogin.com/connector/redash).
+2. Set up Redash callback URL in your SAML provider:
+   `https://app.redash.io/saml/callback?org_slug=<organization name>`. If you
+   have single organization name, set `default` as value for org_slug.
+3. Within SAML provider, map user fields FirstName and LastName to matching
+   values as those two fields are mandatory for user creation.
+
+By default any user created with SAML/SSO will join the default group. It's
+possible to configure the SAML provider to pass what groups the user should join
+by setting the `RedashGroups` parameter. If you use OneLogin's predefined Redash
+application, it will always pass this parameter, meaning that even for existing
+users, it will override their current groups memberships. Hence you need to make
+sure it's up to date.
+
+### How to Configure self-hosted SAML
+
+1. SAML Metadata URL has to look as following:
+   `http://your-site.com/auth/realms/somerelm/protocol/saml/descriptor`
```

Comment:
This isn't strictly true. The metadata URL should point at an XML file. Perhaps update to the following?

```suggestion
1. SAML Metadata URL should point to an XML file on your server, such as:
   `http://your-site.com/auth/realms/somerelm/protocol/saml/descriptor`
```